Makefile

```diff
@@ -40,4 +40,4 @@ gen-ruby: install-buf guard-GOPATH
 	${GOPATH}/bin/buf generate buf.build/open-feature/flagd --template protobuf/buf.gen.ruby.yaml
 
 gen-schema-yaml: install-yq
-	yq eval -j json/flagd-definitions.json | yq -P > json/flagd-definitions.yaml
+	yq eval -o=json json/flagd-definitions.json | yq -P > json/flagd-definitions.yaml
```

Comment:
I wonder if we should consider going the other way. Manually managing a yaml file tends to be easier than json. What do you think?

Reply:
Im happy to make this change, I agree it'll be easier to manage